nav2_theta_star_planner/src/theta_star_planner.cpp

```diff
@@ -45,6 +45,10 @@ void ThetaStarPlanner::configure(
     RCLCPP_WARN(logger_, "Your value for - .how_many_corners  was overridden, and is now set to 8");
   }
 
+  nav2_util::declare_parameter_if_not_declared(
+    node, name_ + ".allow_unknown", rclcpp::ParameterValue(true));
```

Comment:
Need to add to the configuration guide to Theta* in docs new param

Reply:
done here: https://github.com/ros-planning/navigation.ros.org/pull/371